src/App.js

```diff
@@ -1,13 +1,30 @@
-import React from "react";
+import React, {useState,useEffect} from "react";
+import axios from "axios";
 import "./App.css";
+import CreatePhoto from "./Components/photoComponent";
 
 function App() {
+  const [data, setData] = useState({});
+  useEffect(() => {
+    axios.get("https://api.nasa.gov/planetary/apod?api_key=fzliRPzGtv8laLOSxUmx3BaFza3Hdc6nhogBMxba")
+    .then( (response)=> {
+      //Handle sucess
+      console.log(response.data);
+      setData(response.data);
+
+    })
+    .catch(function (error) {
+     // handle error
+      console.log(error);
+    });
+
+  }, []);
+  console.log(data);
   return (
     <div className="App">
-      <p>
-        Read through the instructions in the README.md file to build your NASA
-        app! Have fun 🚀!
-      </p>
+     <h1>Nasa's Image of the Day</h1>
+      <p>Created by Josiah Williams</p>
+      <CreatePhoto obj={data}/>
```

Comment:
You should check if the data isn't empty before passing it into the component
E.g
```
{ data && <CreatePhoto obj={data} /> }
```
To get the through result, set the initial state value of data to null
```
  const [data, setData] = useState(null);

Reply:
It still works

Reply:
I applied the first change. The second change seem to be breaking my code